bin/feature

```diff
@@ -199,6 +199,9 @@ when 'merge'
    exit 1 unless confirm("Merge feature branch named: '#{feature}' ?")
 
    update = Git::submodules_update("get")
+   pull_desc = pull_info[:description].shellescape
+   failure_message = highlight("Merge conflicts detected, merge #{dev_branch} into #{feature} and resolve conflicts.".shellescape)
```

Comment:
That's what this says, I think. The development branch, `dev_branch` is what feature branches are based off of and merged into. `feature` is the feature branch name.